modules/mobile/app/controllers/mobile/v0/messages_controller.rb

```diff
@@ -39,6 +39,7 @@ def create
 
         message_params[:id] = message_params.delete(:draft_id) if message_params[:draft_id].present?
         create_message_params = { message: message_params.to_h }.merge(upload_params)
+        # this reveals the category. is this valuable? jayson just added it, so i assume we must want to know more
         Rails.logger.info('Mobile SM Category Tracking', category: create_message_params.dig(:message, :category))
```

Comment:
Category: data discover
Action: leave for now. we'll deal with data aggregation later. also, this is fairly new so we might still be trying to gather data.